APE24.rst

```diff
@@ -0,0 +1,443 @@
+|                          | is still maintained and in  | updated.                   |
+|                          | what state. We might be able|                            |
+|                          | to work with Astropy (and   |                            |
+|                          | other communities) on what  |                            |
+|                          | this looks like.            |                            |
++--------------------------+-----------------------------+----------------------------+
+
+.. _pyOpenSci Editorial Board: https://www.pyopensci.org/about-peer-review/#our-editorial-board
+.. _Astropy team roles: https://www.astropy.org/team#roles
+
+As of July 2023, Astropy had the following `pre-APE 24 Affiliated Package review guidelines`_
+and followed this `pre-APE 24 Affiliated Package review process, with templates`_.
+
+.. _pre-APE 24 Affiliated Package review guidelines: https://github.com/astropy/astropy-project/blob/1afe5df1728e86dac51b9b87bc30a7c2cdb289ac/affiliated/affiliated_package_review_guidelines.md
+.. _pre-APE 24 Affiliated Package review process, with templates: https://github.com/astropy/astropy-project/blob/1afe5df1728e86dac51b9b87bc30a7c2cdb289ac/affiliated/affiliated_package_review_process.md
+
+Meanwhile, pyOpenSci has its own
+`submission guide for package authors <https://www.pyopensci.org/software-peer-review/how-to/author-guide.html>`_,
+`accompanying GitHub issue templates <https://github.com/pyOpenSci/software-submission/issues/new/choose>`_, and
+`listing of accepted packages <https://www.pyopensci.org/python-packages.html#our-accepted-python-open-source-packages>`_.
+As of July 2023, they are in the process of revamping the submission form to handle
+collaborations with different communities (e.g., Astropy), including but not limited to:
+
+* creating a community-specific label for filtering and link to current reviews
+  for that community; and
+* customizing the editorial bot to add community-specific tags.
+
+Is open review a deal breaker?
+------------------------------
+
+We do not think so! In the pre-APE 24 process, the reviewer was anonymous by default
+because we modeled the process after traditional astronomical journals.
+Even then, the reviewer had the option to reveal themselves if they wanted to.
+Given that the Astropy ecosystem is open-source anyway, it makes sense for us
+to move away from the journal-style tradition and embrace a more open process that
+pyOpenSci uses.
+
+Rejecting packages
+------------------
+
+If a package is "rejected," it is likely not in technical or domain scope.
+The rejection will come from pyOpenSci Editor-in-Chief or the editor;
+it does *not* have to come from the Astropy community directly and will be objective.
+Rejection can also happen if the maintainer of the package under review
+is unwilling to ensure the package can be used and maintained.
+While in practice, rejection is rare, usually the Editor-in-Chief would be
```

Comment:
Can we adjust this just a bit. rejection could happen at varying stages BUT it's most likely it would happen during the pre-review checks that the EIC implements. so you are right that most often the EIC would perform that step. however please also know it could be possible for a package to begin the review process and be rejected as well. and that rejection would come from the editor for that review. 

we haven't had that happen yet. but it could happen. 
Alternatively what can also happen is a package maintainer might stop responding on a review. so we'd close it. i just want to ensure this is clear here but i totally understand what you are saying!

Reply:
Okay, I will rewrite this.